scripts/zones/Beadeaux/Zone.lua

```diff
@@ -50,17 +57,66 @@ zone_object.onConquestUpdate = function(zone, updatetype)
 end
 
 zone_object.onRegionEnter = function(player, region)
-    if region:GetRegionID() <= 6 then
```

Comment:
Why are these quest options being removed or commented out?  It appears that some sort of implementation exists.

Reply:
I was capturing the quest and planned on converting it. I can leave it there for the time being though